### DISCUSSION TITLE
More workstation upgrades

### DISCUSSION THREAD

**Discussion by BadnanaHD** [P3pp3rF1y/SophisticatedBackpacks#471] [Ideas]
I would be cool if more workstation Upgrades would be added
For example:
Anvil
Enchantment Table upgradable with bookshelves
Brewing ( maybe compatible with tank)
And all other would be cool too

**Comment by ClockworkCorvus** [comment]
Bumping this idea. Especially for the brewing stand. It would be really cool to be able to brew potions on the fly and as needed all within your backpack.

**Comment by luan122** [comment]
bump
